.changeset/serious-fishes-collect.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+build(deps-dev): bump eslint from 8.54.0 to 8.57.0
```

Comment:
Should we allow dependabot to update packages solely based on the lock file?

I mean, without changing any `package.json`.

Reply:
I don't think so,  we should using an [increase versioning strategy](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#versioning-strategy)

Reply:
True, it's actually changing the @types from node:20 to node:18.

It's a weird one; let's just close it.